renovate.json

```diff
@@ -11,6 +11,18 @@
       "depNameTemplate": "open-feature/flagd",
       "datasourceTemplate": "github-releases"
     },
+    {
```

Comment:
It probably makes sense to group all the flagd changes into a single PR.

https://docs.renovatebot.com/noise-reduction/#package-grouping

Reply:
When testing this on a local branch all flagd version changes are grouped, as all `regexManagers` listed here are under the same `depNameTemplate`